pkg/scheduler/plugins/capacity/capacity.go

```diff
@@ -138,7 +138,7 @@ func (cp *capacityPlugin) OnSessionOpen(ssn *framework.Session) {
 			allocated.Sub(reclaimee.Resreq)
 			victims = append(victims, reclaimee)
 		}
-		klog.V(4).InfoS("Victims from capacity plugin", "victims", victims, "reclaimer", reclaimer)
+		klog.V(4).Infof("Victims from capacity plugin, victims=%+v reclaimer=%s", victims, reclaimer)
```

Comment:
>`victims=%+v` 

Will this just print according to taskInfo's String method of a slice?

Reply:
> > `victims=%+v`
> 
> Will this just print according to taskInfo's String method of a slice?

Yes, "+v" on slice will call each element String meothd if provide